src/components/Nav/Nav.js

```diff
@@ -1,7 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import UserForm from './UserFormComponent/userForm';
+import { userFormInfo } from './NavData';
+
+import {
+  NavContainer,
+  Navbar,
+  NavLogoBox,
+  NavLogoImg,
+  NavLeft,
+  NavMenuBox,
+  NavMenuList,
+  NavUserBox,
+  NavLogin,
+  NavSignup,
+} from './NavEle';
+
+const Img = `/images/metgo3.png`;
 
 function Nav() {
-  return <div>nav</div>;
+  const [isModalOpen, SetIsModalOpen] = useState(false);
+  const [userFormSelect, setUserFormSelect] = useState('');
+  const [form, setForm] = useState('');
+
+  const loginOrSignup = () => {
+    if (userFormSelect === '로그인') {
+      setForm(userFormInfo[0]);
+    } else {
+      setForm(userFormInfo[1]);
+    }
+  };
+
+  const modalOpen = e => {
+    setUserFormSelect(e.target.name);
+    SetIsModalOpen(true);
+    SetIsModalOpen(true);
+    loginOrSignup();
+  };
+
+  const modalClose = () => {
+    SetIsModalOpen(false);
+    SetIsModalOpen(false);
+  };
+
+  return (
+    <NavContainer>
+      <Navbar>
+        <NavLeft>
+          <NavLogoBox>
```

Comment:
```suggestion
    <Container>
      <Navbar>
        <NavLeft>
          <LogoBox>
```
꼭 Nav라는 네이밍을 다 붙여주지 않아도, 그러니까 다른 컴포넌트에 사용한 스타일드 컴포넌트와 스코프가 겹치지 않기 때문에 이름을 중복해도 상관 없습니다

Reply:
> 꼭 Nav라는 네이밍을 다 붙여주지 않아도, 그러니까 다른 컴포넌트에 사용한 스타일드 컴포넌트와 스코프가 겹치지 않기 때문에 이름을 중복해도 상관 없습니다

넵 ! 이름지옥에서 덕분에 벗어날수있었습니다...